docker-compose.yaml

```
@@ -22,8 +22,7 @@ services:
               capabilities: [gpu]
   copilot_proxy:
     # For dockerhub version
-    # image: moyix/copilot_proxy:latest
-    # command: python3 -m flask run --host=0.0.0.0 --port=5000
+    image: moyix/copilot_proxy:latest
```

Suggestion to comment this out as the compose will fail otherwise. Other option is to comment out the build. 
```suggestion
    # image: moyix/copilot_proxy:latest
```